base_geoengine/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     'name': 'Geospatial support for Odoo',
-    'version': '12.0.1.0.2',
+    'version': '12.0.1.1.0',
```

Comment:
You don't need to change the version. This is done when merged by @OCA-git-bot .